libs/net/elastic/Models/HitModel.cs

```diff
@@ -16,7 +16,7 @@ public class HitModel<T>
     public string Id { get; set; } = "";
 
     [JsonPropertyName("_score")]
-    public float Score { get; set; }
+    public float? Score { get; set; }
```

Comment:
When you sort the score is returned as null.

https://stackoverflow.com/questions/41301691/getting-score-null-in-elastic-search

I don't think this is ideal behaviour.

Reply:
@CelloLiu testing this would have discovered the issue immediately.